language-design.md

```diff
@@ -62,7 +127,22 @@ the commenters?
 
 **Response**
 
+In the comments, it seems that some people are debating the merits of making a language easy to learn vs easy to use, with professional programmers favoring the latter and "everyone else" favoring the former. An example of such an argument is made by Strengthiness 
+> A language like Quorum may be good for learning programming concepts, but as an everyday tool for a professional programmer it would be so overly verbose as to just slow one down
+
+We believe this is a false dichotomy - there isn't necessarily a tradeoff between ease of learning and ease of use for programming languages. Some choices would make a PL harder to use for both first-time users and experienced programmers - consider, for instance, requiring all string literals to be presented as character arrays. I think we can all agree that nobody would prefer having to input
 
+    char myString[] = {'H','i','!','\0'};
+
+to
+
+    char myString[] = "Hi!";
+
+However, in some cases, there may be a tradeoff between making a language easy to learn and easy to use. Scratch, for instance, was designed to be an easy to learn language (since it's meant to be an introductory language for new programmers) but is virtually impossible to write complex programs in. 
+
+We beleive that when making design decisions about a programming language, it's important to let your audience's needs inform your prioritization of making the language easy to learn vs easy to use. It's probably important for some DSLs to be easy to learn, because they may aimed at non-programmers (consider Picobot). 
+
+We do think that it would be valuable to have a general purpose programming language whose focus is on ease of learning, so that inexperienced programmers are also able to develop complex programs (we feel that Python may occupy this space, but if it's not simple enough we're all for someone writing an even easier to pick up language). 
```

Comment:
The predecessor language to Python, ABC, was intended for teaching [[_Wikipedia contributors_, 2015](https://en.wikipedia.org/w/index.php?title=ABC_(programming_language)&oldid=667331843)].